### DISCUSSION TITLE
Simultaneous OpenDTU / Hoymiles DTU Pro?

### DISCUSSION THREAD

**Discussion by bdmartens** [tbnobody/OpenDTU#580] [Q&A]
I've been looking to migrate to OpenDTU from the Hoymiles DTU Pro/Cloud, primarily for improved access to the data.  The distributor my installer got the DTU from didn't purchase/support the direct access MODBUS protocols, it appears, and I've been blocked now at the Hoymiles and distributor level, and my installer is completely unfamiliar with it.

Anyway, so I'm trying to get OpenDTU up and running.  I have a ESP32-WROOM32 (ESP32-D0WDQ5) microcontroller board with the nRF24L01+ expansion and wifi board with external antenna.  I've entered all the serial numbers for the HM-700NT microinverters, and set up all the 360 W panel limits that are attached to each port.

I haven't unplugged my Hoymiles DTU-Pro.  In the same room, the OpenDTU is not apparently receiving any signal.

*** Does the process of receiving a signal require a bidirectional connection to the microinverters - meaning does have one DTU connected prevent a second one from receiving the data? ***

(my other limitation for jumping in with two feet is that I have 13 microinverters, and there appears to be a limitation of 10 in OpenDTU)

If not, then I expect I will need to look into positioning and power filtering approaches as I think I've seen reference to - but I'm open to more suggestions for troubleshooting this.

So far I've had it plugged in for 1h 40min, and no data has been received.

Thanks!

**Comment by bdmartens** [comment]
Some of the console output:
16:35:37.355 > Websocket: [/livedata][3] disconnect
16:35:37.518 > RX Period End
16:35:37.518 > All missing
16:35:37.518 > Nothing received, resend whole request
16:35:37.518 > TX AlarmData Channel: 75 --> 15 72 23 32 27 80 12 49 60 80 11 00 63 E8 34 55 00 00 00 00 00 00 00 00 4E C1 1E 
16:35:38.163 > RX Period End
16:35:38.163 > All missing
16:35:38.163 > Nothing received, resend whole request
16:35:38.163 > TX AlarmData Channel: 3 --> 15 72 23 32 27 80 12 49 60 80 11 00 63 E8 34 55 00 00 00 00 00 00 00 00 4E C1 1E 
16:35:38.783 > RX Period End
16:35:38.783 > All missing
16:35:38.783 > Nothing received, resend count exeeded
16:35:38.835 > TX SystemConfigPara Channel: 23 --> 15 72 23 32 27 80 12 49 60 80 05 00 63 E8 34 55 00 00 00 00 00 00 00 00 5A D5 0A 
16:35:39.078 > RX Period End
16:35:39.078 > All missing
16:35:39.078 > Nothing received, resend whole request
16:35:39.078 > TX SystemConfigPara Channel: 40 --> 15 72 23 32 27 80 12 49 60 80 05 00 63 E8 34 55 00 00 00 00 00 00 00 00 5A D5 0A 
16:35:39.323 > RX Period End
16:35:39.323 > All missing
16:35:39.323 > Nothing received, resend whole request
16:35:39.323 > TX SystemConfigPara Channel: 61 --> 15 72 23 32 27 80 12 49 60 80 05 00 63 E8 34 55 00 00 00 00 00 00 00 00 5A D5 0A 
16:35:39.568 > RX Period End
16:35:39.568 > All missing
16:35:39.568 > Nothing received, resend whole request
16:35:39.568 > TX SystemConfigPara Channel: 75 --> 15 72 23 32 27 80 12 49 60 80 05 00 63 E8 34 55 00 00 00 00 00 00 00 00 5A D5 0A 
16:35:39.812 > RX Period End
16:35:39.812 > All missing
16:35:39.812 > Nothing received, resend whole request
16:35:39.812 > TX SystemConfigPara Channel: 3 --> 15 72 23 32 27 80 12 49 60 80 05 00 63 E8 34 55 00 00 00 00 00 00 00 00 5A D5 0A 
16:35:40.151 > RX Period End
16:35:40.151 > All missing
16:35:40.151 > Nothing received, resend count exeeded
16:35:44.280 > Fetch inverter: 114172233218
16:35:44.280 > Request SystemConfigPara
16:35:44.334 > TX RealTimeRunData Channel: 23 --> 15 72 23 32 18 80 12 49 60 80 0B 00 63 E8 34 5F 00 00 00 00 00 00 00 00 34 A4 2E 
16:35:44.572 > RX Period End
16:35:44.572 > All missing
16:35:44.572 > Nothing received, resend whole request
16:35:44.572 > TX RealTimeRunData Channel: 40 --> 15 72 23 32 18 80 12 49 60 80 0B 00 63 E8 34 5F 00 00 00 00 00 00 00 00 34 A4 2E 
16:35:45.165 > RX Period End
16:35:45.165 > All missing
16:35:45.165 > Nothing received, resend whole request
16:35:45.165 > TX RealTimeRunData Channel: 61 --> 15 72 23 32 18 80 12 49 60 80 0B 00 63 E8 34 5F 00 00 00 00 00 00 00 00 34 A4 2E 
16:35:45.326 > RX Period End
16:35:45.326 > All missing
16:35:45.326 > Nothing received, resend whole request
16:35:45.326 > TX RealTimeRunData Channel: 75 --> 15 72 23 32 18 80 12 49 60 80 0B 00 63 E8 34 5F 00 00 00 00 00 00 00 00 34 A4 2E 
16:35:45.377 > RX Period End
16:35:45.377 > All missing
16:35:45.377 > Nothing received, resend whole request
16:35:45.377 > TX RealTimeRunData Channel: 3 --> 15 72 23 32 18 80 12 49 60 80 0B 00 63 E8 34 5F 00 00 00 00 00 00 00 00 34 A4 2E 
16:35:45.693 > RX Period End
16:35:45.693 > All missing
16:35:45.693 > Nothing received, resend count exeeded
16:35:46.045 > TX AlarmData Channel: 23 --> 15 72 23 32 18 80 12 49 60 80 11 00 63 E8 34 5F 00 00 00 00 00 00 00 00 EE BF F5 
16:35:46.387 > RX Period End
16:35:46.387 > All missing
16:35:46.387 > Nothing received, resend whole request
16:35:46.387 > TX AlarmData Channel: 40 --> 15 72 23 32 18 80 12 49 60 80 11 00 63 E8 34 5F 00 00 00 00 00 00 00 00 EE BF F5 
16:35:47.835 > RX Period End
16:35:47.835 > All missing
16:35:47.835 > Nothing received, resend whole request
16:35:47.835 > TX AlarmData Channel: 61 --> 15 72 23 32 18 80 12 49 60 80 11 00 63 E8 34 5F 00 00 00 00 00 00 00 00 EE BF F5 
16:35:47.843 > RX Period End
16:35:47.843 > All missing
16:35:47.843 > Nothing received, resend whole request
16:35:47.843 > TX AlarmData Channel: 75 --> 15 72 23 32 18 80 12 49 60 80 11 00 63 E8 34 5F 00 00 00 00 00 00 00 00 EE BF F5 
16:35:48.321 > RX Period End
16:35:48.321 > All missing
16:35:48.321 > Nothing received, resend whole request
16:35:48.321 > TX AlarmData Channel: 3 --> 15 72 23 32 18 80 12 49 60 80 11 00 63 E8 34 5F 00 00 00 00 00 00 00 00 EE BF F5 
16:35:48.939 > RX Period End
16:35:48.939 > All missing
16:35:48.939 > Nothing received, resend count exeeded
16:35:49.113 > TX SystemConfigPara Channel: 23 --> 15 72 23 32 18 80 12 49 60 80 05 00 63 E8 34 5F 00 00 00 00 00 00 00 00 FA AB E1 
16:35:49.318 > RX Period End
16:35:49.318 > All missing
16:35:49.318 > Nothing received, resend whole request
16:35:49.318 > TX SystemConfigPara Channel: 40 --> 15 72 23 32 18 80 12 49 60 80 05 00 63 E8 34 5F 00 00 00 00 00 00 00 00 FA AB E1 
16:35:49.478 > RX Period End
16:35:49.478 > All missing
16:35:49.478 > Nothing received, resend whole request
16:35:49.478 > TX SystemConfigPara Channel: 61 --> 15 72 23 32 18 80 12 49 60 80 05 00 63 E8 34 5F 00 00 00 00 00 00 00 00 FA AB E1 
16:35:49.725 > RX Period End
16:35:49.725 > All missing
16:35:49.725 > Nothing received, resend whole request
16:35:49.725 > TX SystemConfigPara Channel: 75 --> 15 72 23 32 18 80 12 49 60 80 05 00 63 E8 34 5F 00 00 00 00 00 00 00 00 FA AB E1 
16:35:49.967 > RX Period End
16:35:49.967 > All missing
16:35:49.967 > Nothing received, resend whole request
16:35:49.967 > TX SystemConfigPara Channel: 3 --> 15 72 23 32 18 80 12 49 60 80 05 00 63 E8 34 5F 00 00 00 00 00 00 00 00 FA AB E1 
16:35:50.168 > RX Period End
16:35:50.168 > All missing
16:35:50.168 > Nothing received, resend count exeeded
16:35:54.282 > Fetch inverter: 114172233744
16:35:54.282 > Request SystemConfigPara
16:35:54.438 > TX RealTimeRunData Channel: 23 --> 15 72 23 37 44 80 12 49 60 80 0B 00 63 E8 34 69 00 00 00 00 00 00 00 00 95 DB 9F 
16:35:54.575 > RX Period End
16:35:54.575 > All missing
16:35:54.575 > Nothing received, resend whole request
16:35:54.575 > TX RealTimeRunData Channel: 40 --> 15 72 23 37 44 80 12 49 60 80 0B 00 63 E8 34 69 00 00 00 00 00 00 00 00 95 DB 9F 
16:35:54.848 > RX Period End
16:35:54.848 > All missing
16:35:54.848 > Nothing received, resend whole request
16:35:54.848 > TX RealTimeRunData Channel: 61 --> 15 72 23 37 44 80 12 49 60 80 0B 00 63 E8 34 69 00 00 00 00 00 00 00 00 95 DB 9F 
16:35:55.065 > RX Period End
16:35:55.065 > All missing
16:35:55.065 > Nothing received, resend whole request
16:35:55.065 > TX RealTimeRunData Channel: 75 --> 15 72 23 37 44 80 12 49 60 80 0B 00 63 E8 34 69 00 00 00 00 00 00 00 00 95 DB 9F 
16:35:55.309 > RX Period End
16:35:55.309 > All missing
16:35:55.309 > Nothing received, resend whole request
16:35:55.309 > TX RealTimeRunData Channel: 3 --> 15 72 23 37 44 80 12 49 60 80 0B 00 63 E8 34 69 00 00 00 00 00 00 00 00 95 DB 9F 
16:35:55.514 > RX Period End
16:35:55.514 > All missing
16:35:55.514 > Nothing received, resend count exeeded
16:35:55.563 > TX AlarmData Channel: 23 --> 15 72 23 37 44 80 12 49 60 80 11 00 63 E8 34 69 00 00 00 00 00 00 00 00 4F C0 44 
16:35:56.205 > RX Period End
16:35:56.205 > All missing
16:35:56.205 > Nothing received, resend whole request
16:35:56.205 > TX AlarmData Channel: 40 --> 15 72 23 37 44 80 12 49 60 80 11 00 63 E8 34 69 00 00 00 00 00 00 00 00 4F C0 44 
16:35:56.850 > RX Period End
16:35:56.850 > All missing
16:35:56.850 > Nothing received, resend whole request
16:35:56.850 > TX AlarmData Channel: 61 --> 15 72 23 37 44 80 12 49 60 80 11 00 63 E8 34 69 00 00 00 00 00 00 00 00 4F C0 44 
16:35:57.495 > RX Period End
16:35:57.495 > All missing
16:35:57.495 > Nothing received, resend whole request
16:35:57.495 > TX AlarmData Channel: 75 --> 15 72 23 37 44 80 12 49 60 80 11 00 63 E8 34 69 00 00 00 00 00 00 00 00 4F C0 44 
16:35:58.142 > RX Period End
16:35:58.142 > All missing
16:35:58.142 > Nothing received, resend whole request
16:35:58.142 > TX AlarmData Channel: 3 --> 15 72 23 37 44 80 12 49 60 80 11 00 63 E8 34 69 00 00 00 00 00 00 00 00 4F C0 44 
16:35:58.882 > RX Period End
16:35:58.882 > All missing
16:35:58.882 > Nothing received, resend count exeeded
16:35:58.929 > TX SystemConfigPara Channel: 23 --> 15 72 23 37 44 80 12 49 60 80 05 00 63 E8 34 69 00 00 00 00 00 00 00 00 5B D4 50 
16:35:59.177 > RX Period End
16:35:59.177 > All missing
16:35:59.177 > Nothing received, resend whole request
16:35:59.177 > TX SystemConfigPara Channel: 40 --> 15 72 23 37 44 80 12 49 60 80 05 00 63 E8 34 69 00 00 00 00 00 00 00 00 5B D4 50 
16:36:00.127 > RX Period End
16:36:00.127 > All missing
16:36:00.127 > Nothing received, resend whole request
16:36:00.127 > TX SystemConfigPara Channel: 61 --> 15 72 23 37 44 80 12 49 60 80 05 00 63 E8 34 69 00 00 00 00 00 00 00 00 5B D4 50 
16:36:00.176 > RX Period End
16:36:00.176 > All missing
16:36:00.176 > Nothing received, resend whole request
16:36:00.176 > TX SystemConfigPara Channel: 75 --> 15 72 23 37 44 80 12 49 60 80 05 00 63 E8 34 69 00 00 00 00 00 00 00 00 5B D4 50 
16:36:00.277 > RX Period End
16:36:00.277 > All missing
16:36:00.277 > Nothing received, resend whole request
16:36:00.277 > TX SystemConfigPara Channel: 3 --> 15 72 23 37 44 80 12 49 60 80 05 00 63 E8 34 69 00 00 00 00 00 00 00 00 5B D4 50 
16:36:00.328 > RX Period End
16:36:00.328 > All missing
16:36:00.328 > Nothing received, resend count exeeded

**Comment by thomasleitner** [comment]
I have the same setup and the same problem here. Did you find a solution?
Thanks!

**Comment by Marc9485** [comment]
OpenDTU or AhoyDTU doesn't work togehter with DTu lite and pro. You can only connect one device with your converter

**Reply by thomasleitner** [reply]
Ok, thanks. This is disappointing ... I wanted to run both in parallel.

**Reply by Marc9485** [reply]
Your welcome. 
Found this in "discussions".
Maybe this will help you:

https://github.com/tbnobody/OpenDTU/discussions/1474

**Reply by Fzsfazer1979** [reply]
J'utilise open dtu et dtu pro en même temps depuis toujours et je n'est jamais eu de problème. Sans faire de réglage particulier

**Comment by bloublou2014** [comment]
Same experience as [Fzsfazer1979](https://github.com/Fzsfazer1979):  i installed an openDTU with HM microinverter (6xHM-700 + 1xHM-1500) alongside a DTUPro and both work with no issue. 
Note: the setup was DTUPro then later on openDTU. openDTU used came from [blinkypart](https://shop.blinkyparts.com/en/OpenDTU/)
